### PR TITLE
fixing dotnet core version issue for resource `azurerm_windows_web_app`

### PR DESCRIPTION
**Pull request by xiaxyi**
The dotnet core web app version is not controlled by the property .NETFrameworkVersion, thus, not to set the version to an unsupported .net framework version such as v3.0

fixing the issue: #16867 #17232 https://github.com/hashicorp/terraform-provider-azurerm/issues/16222

### CODE REVIEW

**Comment by JohnBoneJones** [comment]
May I set current_stack  = "dotnetcore", dotnet_version = "core3.1" in next version?

**Comment by JohnBoneJones** [comment]
How can I set  dotnet_version = "core3.1" for azurerm_windows_web_app

**Comment by xiaxyi** [comment]
> May I set current_stack = "dotnetcore", dotnet_version = "core3.1" in next version and How can I set dotnet_version = "core3.1" for azurerm_windows_web_app?

For asp.net runtime stack, you don't need to set the dotnet_version to core 3.1 as there is no 3.1 version of CLR (you can refer to ). Core3.1 is provided for dotnet core runtime. you can refer to this doc for the version and clr dependency:https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/versions-and-dependencies

File: internal/services/appservice/helpers/web_app_schema.go [hunk 1]
```diff
@@ -879,7 +879,8 @@ func windowsApplicationStackSchema() *pluginsdk.Schema {
 					Type:     pluginsdk.TypeString,
 					Optional: true,
 					ValidateFunc: validation.StringInSlice([]string{
-						"v3.0",
```
Comment (katbyte) [inline internal/services/appservice/helpers/web_app_schema.go:882]: i'm afraid we can't just remove this value, could we add it back in and conditionally remove it when the 4.0 flag is enabled? thanks

**Review by katbyte — CHANGES_REQUESTED** [review]

**Comment by xiaxyi** [comment]
@katbyte Thanks for the comments, changed the code.

**Review by katbyte — APPROVED** [review]
Thanks @xiaxyi - LGTM 🦕

**Pull request merged**